src/test/java/codeu/model/store/basic/MessageStoreTest.java

```diff
@@ -60,6 +61,19 @@ public void testGetMessagesInConversation() {
     assertEquals(MESSAGE_ONE, resultMessages.get(0));
     assertEquals(MESSAGE_TWO, resultMessages.get(1));
   }
+  
+  @Test
+  public void testGetRecentMessages() {
+	Message inputMessage =
```

Comment:
Also add another message to the store which is more than a day ago. We want to make sure that one doesn't show up in the final list.